From ac8ae924ca024b4b56555e8fc032ba02dc27383b Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Tue, 22 Aug 2023 13:26:29 -0500
Subject: [PATCH 26/57] docs: Add menu to prevent older makeinfo versions from
 failing

It has been reported that makeinfo version 4.13a complains and returns
error when menus for chapter structuring commands are not present. It
is also known that newer makeinfos, such as version 6.7, will create
default menus when needed. Since the menu will be created regardless,
explicitly create it to support older makeinfo versions. This also
enables building to be successful when an older makeinfo is installed
because in that case info files are attempted to be generated with the
"all" target.

Reported-by: Olaf Hering <olaf@aepfle.de>
Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
Tested-by: Olaf Hering <olaf@aepfle.de>
---
 docs/grub-dev.texi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/docs/grub-dev.texi b/docs/grub-dev.texi
index a695b02f0..1276c5930 100644
--- a/docs/grub-dev.texi
+++ b/docs/grub-dev.texi
@@ -788,6 +788,11 @@ out of its way to avoid using hardware breakpoints internally and uses them as
 briefly as possible when needed, thus allowing the user to have a maximal
 number at their disposal.
 
+@menu
+* OVMF debug log::
+* Using the gdbinfo command::
+@end menu
+
 @node OVMF debug log
 @subsection OVMF debug log
 
-- 
2.42.0

